docs/src/main/tut/json.md

```diff
@@ -31,9 +33,9 @@ community.  The functionality is similar:
 
 ```scala
 libraryDependencies += Seq(
-  "org.http4s" %% "http4s-argonaut" % "{{< version >}}",
+  "org.http4s" %% "http4s-argonaut" % http4sVersion,
   // Optional for auto-derivation of JSON codecs
-  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M4"
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "{{< version "argonaut-shapeless_6.2" >}}"
```

Comment:
I was going to remove the `argonaut-shapeless_6.2` version from `build.sbt`/`build.toml` on `master` until I realized it was still cited in the JSON tut.

We're now on latest milestone tag of argonaut-shapeless, `1.2.0-M5`, but I don't understand the broader landscape to know if that's appropriate with http4s on fs2.